app/test_main.py

```diff
@@ -1,3 +1,20 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,result",
+    [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
```

Comment:
It seems like the test case for (14, 14, [0, 0]) and (15, 15, [1, 1]) might be testing similar conditions. Ensure that each test case is unique and tests a different scenario to avoid repetition, as per the 'Don't Repeat Yourself' checklist item.